tests/pauli/dla/test_structure_constants.py

```diff
@@ -84,12 +84,13 @@ def test_structure_constants_elements_with_orthogonal_basis(
 
     @pytest.mark.parametrize("ortho_dla", [Ising3, XXZ3])
     @pytest.mark.parametrize("matrix", [False, True])
-    def test_structure_constants_elements_with_non_orthogonal(self, ortho_dla, matrix):
+    def test_structure_constants_elements_with_non_orthogonal(self, ortho_dla, matrix, seed):
         r"""Test relation :math:`[i G_α, i G_β] = \sum_{γ=0}^{d-1} f^γ_{α,β} iG_γ_` with
         non-orthogonal bases.
         """
         d = len(ortho_dla)
 
+        np.random.seed(seed)
         coeffs = np.random.random((d, d)) + 0.5
```

Comment:
```suggestion
        rng = np.random.default_rng(seed)
        coeffs = rng.random((d, d)) + 0.5
```
Don't want to interfere with other tests.

Reply:
ah nice! didnt know this in regular numpy, just from jax